admin/views/form/select.php

```diff
@@ -18,7 +18,11 @@
 }
 
 ?>
-<select <?php echo $attributes; ?>name="<?php echo esc_attr( $name ); ?>" id="<?php echo esc_attr( $id ); ?>">
```

Comment:
Suggestion: leave the HTML line as is and add the following on the line above it:
<?php /* phpcs:ignore .... */ ?>